test/Parser.js

```diff
@@ -486,8 +486,14 @@ describe('Parser', () => {
         div {
           ${'width:0;    '}
         }
+        .foo {
```

Comment:
Are we missing the test for `div :hover`?

Reply:
no, it's here: https://github.com/thysultan/stylis.js/pull/190/files#diff-2c21dd491e96fcd23ee0c155c2141f07R212

but i can move it from a separate test to this "whitespace" test, if we do this then the second similar test for `& :hover` should be moved as well, although it's now basically the same situation as `div :hover`